nixos/modules/installer/tools/nixos-enter.sh

```diff
@@ -32,9 +33,12 @@ while [ "$#" -gt 0 ]; do
             exit 1
             ;;
         --command|-c)
-            command=($system/sw/bin/bash "-c" "$1")
+            command=("$system/sw/bin/bash" "-c" "$1")
             shift 1
             ;;
+	--silent)
```

Comment:
I might have messed a tab in here.

Reply:
Feel free to rebase.

Reply:
If possible, please squash all commits into one.

Reply:
Sure thing. I thought you were going to do this on GitHub.